src/Export/Field/FilterAttributes.php

```diff
@@ -58,7 +62,7 @@ private function getSelectedFilterAttributes(Article $article): string
         $result = '';
         foreach (array_intersect_key(parent::getData($article), $this->getMultiAttributes()) as $key => $value) {
             if ($value) {
-                $result .= $this->filter->filterValue($key) . '=' . $this->filter->filterValue((string) $value) . '|';
+                $result .= $this->filter->filterValue($key) . '=' . $this->filter->filterValue((string)$value) . '|';
```

Comment:
can you add this space again? AFAIR we always have bee n using the spacing between value and casted types. Unless you know a PSR which tells different story